recipe/meta.yaml

```diff
@@ -37,17 +39,24 @@ outputs:
         # The old RTM API has a dependency on aiohttp
         # - slack_sdk.rtm
         - slack_sdk.rtm_v2
+      requires:
+        - pip
+        - aiohttp >=3.7.3,<4
       commands:
         - pip check
-        - conda install -y aiohttp
         - python -c "import slack_sdk.rtm"
```

Comment:
Could this line be moved to imports now that you added `aiohttp` to the requires section?

Reply:
Thanks. I've moved it to imports and clarified a comment